charts/prefect-orion/values.yaml

```diff
@@ -32,10 +32,12 @@ orion:
     # -- enable orion image debug mode
     debug: false
 
+  # -- PREFECT_ORION_UI_API_URL; should be publically accesible API url; default to PREFECT_API_URL for running API locally
+  publicApiUrl: ""
+
   # -- array with environment variables to add to orion nodes
   env: []
   ## env:
-  ##   -- API url used by services such as the agent; can be in-cluster API url
   ##   - name: PREFECT_API_URL
   ##     value: http://prefect-orion:4200/api
```

Comment:
What nodes would these be? A better example setting may be changing the logging level.